Lib/shutil.py

```diff
@@ -701,6 +701,8 @@ def onerror(*args):
             os.close(fd)
     else:
         try:
+            if not os.path.exists(path):
+                raise FileNotFoundError("cannot call rmtree on a missing path")
```

Comment:
I think it would be better replace 'cannot' with 'Cannot' and add a full stop after 'path'. Thus:

`Cannot call rmtree on a missing path.`

Reply:
I updated the capitalization but not the punctuation to remain consistent with the other error messages in the function.